lib/models/user.rb

```diff
@@ -23,4 +31,21 @@ def self.random
       }
     )
   end
+
+  def self.gen_file(opts = {})
+    rows = 0
+    rows = opts[:rows] if opts[:rows]
+    users = []
+    if(opts[:rows])
+      rows.times do |x|
+        users.push(User.random)
+      end
+    end
+    CSV.open("./users.csv", "wb") do |csv|
+      csv << ["name", "sis_id", "login_id", "email", "time_zone"]
```

Comment:
this works to write headers CSV also takes an option to pass :write_headers (boolean) and :headers(array).
e.g `
header = ["name", "sis_id", "login_id", "email", "time_zone"]
CSV.open("./users.csv", "a", write_headers:true, headers:header) do
        end`
a in this case will just write to the end of the file which if we want to just create a new file each time then wb is best.
wb will truncate the existing file and sets the external encoding to ascii-8bit not sure if that will matter for our case or if we would want to set the encoding to utf-8 if so then we just have to pass :encoding and the specific one

Reply:
yes, always renew the file, but I'll update the headers